autodepgraph/node.py

```diff
@@ -27,8 +28,15 @@ def __init__(self, name, verbose=False):
                            parameter_class=ManualParameter,
                            vals=vals.Numbers(min_value=0))
 
-        self.add_parameter('dependencies',
-                           docstring='a list of names of Calibration nodes',
+        self.add_parameter('parents',
+                           docstring='List of names of nodes '
+                                     + 'on which this node depends',
+                           initial_value=[],
+                           vals=vals.Lists(vals.Strings()),
+                           parameter_class=ManualParameter)
```

Comment:
I would not make this a manual parameter as it should only be modified using the add/remove parent functions. I would set a get_cmd to return the self._parents and make the docstring refer to the add/remove parent functions. 
Alternatively (and maybe even better) you could support setting a list of strings but make sure that the set_cmd iterates over the elements in the list using the add parent function.